README.md

```diff
@@ -1,118 +1,44 @@
-# Barnardos consent form generator
+# consent-form-builder
```

Comment:
This seems like a regression from the previous title.

Reply:
The aim of these `README.md` updates was to bring it inline with our other repos' `README.md`s. Our other repos have the repo name as the heading for the document.